src/components/picker/index.js

```diff
@@ -233,6 +242,7 @@ class Picker extends BaseComponent {
           getItemValue: child.props.getItemValue || getItemValue,
           onSelectedLayout: this.onSelectedItemLayout,
           renderItem: child.props.renderItem || renderItem,
+          accessibilityHint: 'Double click to select this suggestion'
```

Comment:
... to select this option

Reply:
it's not my content, it's according to the UX guidelines